src/main/java/com/couchbase/lite/support/Batcher.java

```diff
@@ -167,6 +166,17 @@ public int count() {
         }
     }
 
+    /**
+     * this is mainly for debugging
+     */
+    public int sizeOfPendingFutures() {
+        synchronized(this) {
+            if(pendingFutures == null) {
+                return 0;
+            }
+            return pendingFutures.size();
+        }
+    }
 
```

Comment:
Are we using this in uni tests or an hoc debugging? I think we just clean it out if it's just for one-time debugging.